File: easy_33_sum_of_array_of_5_integers/solution.py
from typing import TextIO
import sys

class Solution:
    def sumOfFive(self, arr):
        """
        Returns the sum of an array of 5 integers.
        """
        return sum(arr)

def solve(infile: TextIO = sys.stdin, outfile: TextIO = sys.stdout) -> None:
    arr = list(map(int, infile.read().strip().split()))
    solution = Solution()
    result = solution.sumOfFive(arr)
    outfile.write(f"{result}\n")

if __name__ == "__main__":
    solve()

File: easy_33_sum_of_array_of_5_integers/solution.cpp
using namespace std;
#include <vector>
#include <iostream>

class Solution {
public:
    int sumOfFive(const std::vector<int>& arr) {
        int total = 0;
        for (int x : arr) total += x;
        return total;
    }
};

int main() {
    std::vector<int> arr(5);
    for (int i = 0; i < 5; ++i) {
        std::cin >> arr[i];
    }
    Solution solution;
    int result = solution.sumOfFive(arr);
    std::cout << result << std::endl;
    return 0;
}